src/can-map-attributes.js

```diff
@@ -1,17 +1,14 @@
-var can = require('can/util/util');
-require('can/map/map');
-require('can/map/map_helpers');
-require('can/list/list');
+var can = require('can-util');
+var Map = require('can-map');
+require('can-map/map-helpers');
+require('can-list');
 
 //!steal-remove-start
 can.dev.warn("can/map/attributes is a deprecated plugin and will be removed in a future release. "+
 	"can/map/define provides the same functionality in a more complete API.");
 //!steal-remove-end
 
-can.each([
-	can.Map,
-	can.Model
-], function (clss) {
+function applyAttributes (clss) {
```

Comment:
I would just import `can-model`.  The import trick is smart, but as there's no way to control dependency order, it's possible this might be loaded first.  In the vast majority of legacy apps that upgrade, they will probably be using both Map and Model. So we aren't really likely loading stuff that they dont need.